pkgs/swift2objc/lib/src/ast/declarations/built_in/built_in_declaration.dart

```diff
@@ -2,9 +2,15 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
+import '../../../parser/_core/utils.dart';
+import '../../../transformer/transform.dart';
```

Comment:
The overall approach of this looks fine, but the AST shouldn't be importing things from the transformer. Can you move the `primitiveWrappers` stuff to the transformer folder?